release-smokeping.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/sh
```

Comment:
changing my release process is not ideal :) could you create a separate PR for that

Reply:
Seems I missed replacing your version before opening the pull request my bad.